lib/inherited_resources/responder.rb

```diff
@@ -3,5 +3,10 @@
 module InheritedResources
   class Responder < ActionController::Responder
     include Responders::FlashResponder
+    include Responders::HttpCacheResponder
```

Comment:
This doesn't appear to be a new default but [was always part of the generator for over a decade](https://github.com/heartcombo/responders/commit/4d094297022f6ab6db6d4a9973dc7453133c1f1c#diff-544fda81eeab7956b30883ad942d73dab5ba15b53c593f19dfd6e892680e85b4).

Based on [the class documentation details](https://github.com/heartcombo/responders/blob/main/lib/responders/http_cache_responder.rb#L4-L12) and also [what the code does internally](https://github.com/heartcombo/responders/blob/main/lib/responders/http_cache_responder.rb#L27-L29), I do not want to introduce this as a default here. Would be safest to just focus on only the Turbo related changes.

```suggestion
```

Reply:
I do agree. I've asked to remove this in the other PR, but then I've seen it was a default and reintroduce it.

I will add a note in the commit that we are deliberately not including that line